lib/grpperm.gi

```diff
@@ -1232,9 +1232,72 @@ end );
 ##
 #M  IsSolvableGroup( <G> )  . . . . . . . . . . . . . . . .  solvability test
 ##
+
+# fast test for finding iteratively nontrivial commutators in a permutation group.
+# if this goes on too long the group may not be solvable. This can be much faster
+# for large degree groups than to use the full pcgs machinery (which uses the same
+# idea but builds a pcgs on the way).
+BindGlobal("QuickUnsolvabilityTestPerm",function(G)
+local som,elvth,fct,gens,new,l,i,j,a,b,bound;
+  # a few moved points
+  som:=MovedPoints(G);
+  bound:=Int(LogInt(Length(som)^5,3)/2);
```

Comment:
Is there a deeper mathematical reason for this bound, or is it just a heuristic?

Reply:
This is (written slightly awkwardly, as we don't have a root) bound for solvable length of a permutation group due to Dixon. I add a comment.